imgui should not be built for Athena

**Describe the bug**
imgui shouldn't be built for the Athena (roboRIO) platform, since the gui tools aren't built for it. I get compile errors when building on windows

**To Reproduce**
Steps to reproduce the behavior:
1. run `gradlew build` on windows, with the athena cross compiler installed

**Expected behavior**
A clear and concise description of what you expected to happen.

**Screenshots**
Example of compile failures when running `gradlew build` on windows
```
> Task :thirdparty:imgui_suite:compileImguiLinuxathenaDebugStaticLibraryImguiC
In file included from C:\Users\chris\Documents\Robotics\allwpilib\thirdparty\imgui_suite\glfw\src\platform.h:62,
                 from C:\Users\chris\Documents\Robotics\allwpilib\thirdparty\imgui_suite\glfw\src\internal.h:331,
                 from C:\Users\chris\Documents\Robotics\allwpilib\thirdparty\imgui_suite\glfw\src\null_init.c:30:
C:\Users\chris\Documents\Robotics\allwpilib\thirdparty\imgui_suite\glfw\src\x11_platform.h:32:10: fatal error: X11/Xlib.h: No such file or directory
   32 | #include <X11/Xlib.h>
      |          ^~~~~~~~~~~~
compilation terminated.
```

**Desktop (please complete the following information):**
 - OS: Windows 11

--- a/thirdparty/imgui_suite/build.gradle
+++ b/thirdparty/imgui_suite/build.gradle
@@ -24,7 +24,6 @@ nativeUtils.platformConfigs.named('osxuniversal') {
 }
 
 model {
-
     components {
         imgui(NativeLibrarySpec) {
             sources {
@@ -50,6 +49,10 @@ model {
                 }
             }
             binaries.all {
+                if (it.targetPlatform.name == nativeUtils.wpi.platforms.roborio) {
+                    it.buildable = false
+                    return
+                }
                 if (toolChain in VisualCpp) {
                     cppCompiler.args '-D_UNICODE', '-DUNICODE', '-DWIN32', '-D_WIN32', '-DSTRICT', '-DWIN32_LEAN_AND_MEAN', '-D_HAS_EXCEPTIONS=1'
                 } else {

